sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -237,6 +229,9 @@ case class Divide(left: Expression, right: Expression)
     }
   }
 
+  // Used by doGenCode
+  protected def divide(eval1: ExprCode, eval2: ExprCode, javaType: String): String
```

Comment:
@cloud-fan yes but getting ```A method named "$div" is not declared in any enclosing class nor any supertype, nor through a static import``` in the updated pr for `Code generation of (2.0 / 1.0)`